README.md

```diff
@@ -93,11 +103,34 @@ link against.
 Once OpenSSL is set up, you need to configure omsagent to include the
 ```--enable-ulinux``` qualifier, like this:<br>```./configure --enable-ulinux``` 
 
-### To check out source code repository
+#### Preparing system tests
+**This section only aplies if you do not have write access to MSFTOSSMgmt.**
 
-To check out the source code repository, you need a command like:
+System tests communicate with the server. They require keys which may not be published.
+You may obtain your own keys for free by creating an account on
+[Microsoft Operations Management Suite](http://www.microsoft.com/en-us/server-cloud/operations-management-suite/why-oms.aspx)
+then get the keys from the settings view.
 
-```git clone --recursive git@github.com:MSFTOSSMgmt/bld-omsagent.git```
+Create the file `bld-omsagent/omsagent/test/config/systest.conf`
+with content like so:
+``` shell
+export TEST_WORKSPACE_ID=<id>
+export TEST_SHARED_KEY=<key>
+
+export TEST_WORKSPACE_ID_2=<id>
+export TEST_SHARED_KEY_2=<key>
+
+export TEST_PROXY_SETTING="http://proxyuser:proxypass@host:8080"
+```
+
+### Cloning the repositories
+
+If you have write access to MSFTOSSMgmt use the following. Otherwise, refer to [Preparing system tests](#preparing-system-tests).
```

Comment:
That's not quite right. If they don't have <b>read</b> access to MSFTOSSMgmt group, then they do the first clone statement, then create the file IF they care for successful unit tests.

Clarify this. If they're not interested in successful system tests, they can continue with the first clone statement and then continuing along their way.